app/src/main/res/values/strings.xml

```diff
@@ -998,9 +998,10 @@ A level counts as a roof level when its windows are in the roof. Subsequently, r
     <string name="quest_crossing_kerb_height_title">What’s the height of the curbs at this crossing?</string>
 
     <string name="quest_crossing_title2">Is there a crossing here?</string>
-    <string name="quest_crossing_yes">Yes (e.g. the curb is lowered here or there are markings)</string>
-    <string name="quest_crossing_no">No, but crossing is possible</string>
+    <string name="quest_crossing_yes2">Yes, intended as crossing</string>
+    <string name="quest_crossing_no2">Not intended as crossing, still possible to cross (e.g. informal trails)</string>
```

Comment:
Is it correct to change the "name" field because I want to force the translations to also change their wording?

Is it correct to remove the old translation strings?

Reply:
Hm well, if the wording changes substantially, then yes. But in this case, IMO, it doesn't change substantially, so the name should not be changed.

When the source translation changes, existing translations of this string are shown as "fuzzy" in POEditor (the crowd translation platform), it is also possible to filter showing only fuzzy strings.

Reply:
Ah, I see https://github.com/streetcomplete/StreetComplete/pull/5936/commits/e9b9b93489a2e653aec58d8dc01d842f3aa14648